setup.cfg

```
@@ -24,7 +24,7 @@ install_requires =
     nibabel >=2.2.1
     nipype >=1.3.1
     nitime
-    niworkflows ~= 1.0.3
+    niworkflows ~= 1.1.3
```

```suggestion
    niworkflows ~= 1.0.4
```

Do not commit until https://pypi.org/project/niworkflows/1.0.4/ exists.